server.js

```diff
@@ -0,0 +1,10 @@
+const server = require("express")();
+const path = require("path");
+const serveStatic = require("serve-static");
+
+server.use(serveStatic(path.join(`${__dirname}/dist`)));
+
+const port = process.env.PORT || 5000;
+server.listen(port);
+
+console.log(`Begin your campaign on http://localhost:${port}`);
```

Comment:
Quick little server to serve our dist folder (hopefully created by the post-install script on package.json) to heroku

Reply:
Very neat, like we are bypassing the need to build all of the complex files